assets/pages/regular-contributions-existing/regularContributionsExisting.jsx

```diff
@@ -36,13 +34,8 @@ const content = (
           Guardian - thank you. If you&#39;re feeling generous, there is
           another way you can&nbsp;help.
       </p>
-      <CtaLink
-        ctaId="contribute-one-off-again"
-        text="Make a one-off contribution"
-        url={routes.oneOffContribCheckout}
-        accessibilityHint="Further support the guardian over and above your current regular contribution"
-      />
-    </PageSection>
+      <ManageContributionCta />
+    </PageSection >
```

Comment:
I don't fully understand why, but github's parser is erroring this line.

Reply:
I think it's the trailing space between the word `PageSection` and the closing `>`,

Reply:
Ahhhh thanks!